Feat: update testOptions (#2147)
saucelabs/sauce-docs: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

---

Sauce Labs runs your tests using [Android Debug Bridge (ADB)](https://developer.android.com/studio/test/command-line#run-tests-with-adb) for Android Real Devices and Android Emulators by invoking the `adb shell am instrument`-command. This provides a set of [instrumentation options](https://developer.android.com/studio/test/command-line#am-instrument-options) that you can use to control the test execution. We offer support for predefined and custom options.

:::info
Available options:
- optional and can be used in any combination unless otherwise noted
- can be used on both Android Emulators and Real Devices unless otherwise noted with <span className="sauceGreen">Virtual Devices Only</span> or <span className="sauceGreen">Real Devices Only</span>

:::

### `testOptions`

<p><small>| OPTIONAL | OBJECT |</small></p>

> The `testOptions` property allows you to provide options to `saucectl`. It's a set of parameters allowing you to provide additional details about which test class should be run for the suite and how to apply them. More information can be found in the official Android ["Test from the command line"](https://developer.android.com/studio/test/command-line#am-instrument-options) and ["AndroidJUnitRunner"](https://developer.android.com/reference/androidx/test/runner/AndroidJUnitRunner) docs.

Reviewer: ```suggestion
The `testOptions` property allows you to provide options to `saucectl`. It's a set of parameters allowing you to provide additional details about which test class should be run for the suite and how to apply them. For more information, see the official Android ["Test from the command line"](https://developer.android.com/studio/test/command-line#am-instrument-options) and ["AndroidJUnitRunner"](https://developer.android.com/reference/androidx/test/runner/AndroidJUnitRunner) docs.
```

---

> Sauce Labs runs your tests using [Android Debug Bridge (ADB)](https://developer.android.com/studio/test/command-line#run-tests-with-adb) for Android Real Devices and Android Emulators by invoking the `adb shell am instrument`-command. This provides a set of [instrumentation options](<[``](https://developer.android.com/studio/test/command-line#am-instrument-options)>) that you can use to control the test execution. We offer support for predefined and custom options.

Reviewer: ```suggestion
Sauce Labs runs your tests using [Android Debug Bridge (ADB)](https://developer.android.com/studio/test/command-line#run-tests-with-adb) for Android Real Devices and Android Emulators by invoking the `adb shell am instrument`-command. This provides a set of [instrumentation options](https://developer.android.com/studio/test/command-line#am-instrument-options) that you can use to control the test execution. We offer support for predefined and custom options.
```

> :::info
> All options are:
> 
> - optional and can be used in any combination unless otherwise noted
> - can be used on both Android Emulators and Real Devices unless otherwise noted with <span className="sauceGreen">Virtual Devices Only</span> or <span className="sauceGreen">Real Devices Only</span>
> 
> :::

Reviewer: ```suggestion
:::info
Available options:
- optional and can be used in any combination unless otherwise noted
- can be used on both Android Emulators and Real Devices unless otherwise noted with <span className="sauceGreen">Virtual Devices Only</span> or <span className="sauceGreen">Real Devices Only</span>

:::
```
